CHANGES

```diff
@@ -9,6 +9,15 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) All of the low level cipher functions have been deprecated including:
+     IDEA_encrypt, IDEA_options, IDEA_ecb_encrypt, IDEA_set_encrypt_key, IDEA_set_decrypt_key, IDEA_cbc_encrypt, IDEA_cfb64_encrypt, IDEA_ofb64_encrypt,
```

Comment:
Probably the existing entries for Blowfish, AES, etc can be merged into this one.

Reply:
Okay, will do in the morning.